ui/apps/platform/src/Containers/Vulnerabilities/utils/searchUtils.tsx

```diff
@@ -16,6 +16,7 @@ import { getQueryString } from 'utils/queryStringUtils';
 import { searchValueAsArray, getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
 import { ensureExhaustive } from 'utils/type.utils';
 
+import { ensureStringArray } from 'Components/CompoundSearchFilter/utils/utils';
```

Comment:
It looks like you will need to rebase after Saif merges this refactoring:
https://github.com/stackrox/stackrox/pull/12810/files#diff-193f7a8c5f64fc3c0c2f57df54a36d3bda3e001973dae858a5134bdfe181967eR31